docs/GettingStarted.md

```diff
@@ -104,7 +104,7 @@ module.exports = api => {
 };
 ```
 
-> Note: `babel-jest` is automatically installed when installing Jest and will automatically transform files if a babel configuration exists in your project. To avoid this behavior, you can explicitly reset the `transform` configuration option:
+:::tip `babel-jest` is automatically installed when installing Jest and will automatically transform files if a babel configuration exists in your project. To avoid this behavior, you can explicitly reset the `transform` configuration option :::
```

Comment:
this line is making the netlify deploy preview and the reason is prettier as it tries to make it one line and the Admonitions fails to work